mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQ.kt

```diff
@@ -0,0 +1,7 @@
+package org.mifos.mobilewallet.mifospay.faq.ui
+
+data class FAQ(
+    var question: String? = null,
```

Comment:
question will be first point of rendering faq list so It can't be null but answer can be null or empty